docs/sdk-and-tools/erdjs/migration-guides.md

```diff
@@ -9,9 +9,140 @@ This tutorial will guide you through the process of migrating from one major ver
 Make sure you have a look over the [cookbook](/sdk-and-tools/erdjs/erdjs-cookbook), in advance.
 :::
 
+## Migrate **erdjs** from v11.x to v12 (November of 2022)
+
+**erdjs v12** brings a series of breaking changes that we will explain below.
+
+### Transaction version and options
+
+Starting with **erdjs v12**, support for `GuardAccount` feature will be available to use. This means that another option for transaction has been activated. Therefore, now we have: 
+
+```
+export const TRANSACTION_OPTIONS_DEFAULT = 0;
+export const TRANSACTION_OPTIONS_TX_HASH_SIGN = 1;
+export const TRANSACTION_OPTIONS_TX_GUARDED = 2;
+```
+Given this addition it means that previously `export const TRANSACTION_VERSION_TX_HASH_SIGN = 2` becomes obsolete because a *version 2* transaction will support more than one option. That's why we changed this constant to:
+
+```
+export const TRANSACTION_VERSION_WITH_TX_OPTIONS = 2;
```

Comment:
`TRANSACTION_VERSION_WITH_OPTIONS`? So we don't have the word transaction duplicated?

Reply:
Changed it also in `elrond-sdk-erdjs`